dev/yarn.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM yarnpkg/node-yarn:latest
+
+RUN mkdir /web
+WORKDIR /web
+COPY ./web /web
+RUN npm cache clean -f
+RUN npm install n -g -q
+RUN n stable -q
```

Comment:
I think it makes more sense to use a base image with the correct Node.js version rather than use the [`n` tool](https://www.npmjs.com/package/n).

Reply:
After my discussion with Daniel, it seems that for development, the better approach is to mount the local copy of the `node_modules` directory instead of using a dockerfile to build and install them separately inside the container. I changed this in [1dd90b0](https://github.com/atlascope/atlascope/pull/136/commits/1dd90b08f7ff7e536999263030583820c9eb0bd3).